### PR TITLE
Revise "rmsprop_vn" 

### LINKED ISSUE #2070: Revise "rmsprop_vn"
Phần này được dịch bởi: **davidnvq**
Nếu bạn đã dịch phần này, vui lòng bỏ qua việc revise.

### PR DESCRIPTION
**Pull request by quangnhat185**
Close #2070

### CODE REVIEW

File: chapter_optimization/rmsprop_vn.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ An alternative is to use a leaky average in the same way we used in the momentum
...
 -->
 
-Vấn đề này có thể được giải quyết bằng cách sử dụng $\mathbf{s}_t / t$.
-Đối với các phân phối hợp lý của $\mathbf{g}_t$, thuật toán sẽ hội tụ.
-Đáng tiếc là có thể mất rất nhiều thời gian cho đến khi giới hạn bắt đầu ảnh hưởng, vì thuật toán này ghi nhớ toàn bộ quỹ đạo của các giá trị.
-Một cách khác là sử dụng trung bình rò rỉ tương tự như trong phương pháp động lượng, tức là $\mathbf{s}_t \leftarrow \gamma \mathbf{s}_{t-1} + (1-\gamma) \mathbf{g}_t^2$ cho các tham số $\gamma > 0$.
-Giữ nguyên tất cả các phần khác ta có thuật toán RMSProp.
+Vấn đề này có thể được giải quyết bằng cách sử dụng $\mathbf{s}_t / t$. 
+Đối với các phân phối hợp lý của $\mathbf{g}_t$, thuật toán sẽ hội tụ. 
+Đáng tiếc là có thể mất rất nhiều thời gian cho đến khi trạng thái giới hạn được bắt đầu, bởi thuật toán này ghi nhớ toàn bộ quỹ đạo của các giá trị.
```
Comment (lkhphuc) [inline chapter_optimization/rmsprop_vn.md:40]:
```suggestion
Đáng tiếc là có thể mất rất nhiều thời gian cho đến khi các tính chất tại giới hạn bắt đầu có ảnh hưởng, bởi thuật toán này ghi nhớ toàn bộ quỹ đạo của các giá trị. 
```
Chỗ này mình dịch hơi thoáng, mình hiểu `limit behavior` là các tính chất mà nó có khi hội tụ ("giới hạn").

File: chapter_optimization/rmsprop_vn.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ An alternative is to use a leaky average in the same way we used in the momentum
...
-Vấn đề này có thể được giải quyết bằng cách sử dụng $\mathbf{s}_t / t$.
-Đối với các phân phối hợp lý của $\mathbf{g}_t$, thuật toán sẽ hội tụ.
-Đáng tiếc là có thể mất rất nhiều thời gian cho đến khi giới hạn bắt đầu ảnh hưởng, vì thuật toán này ghi nhớ toàn bộ quỹ đạo của các giá trị.
-Một cách khác là sử dụng trung bình rò rỉ tương tự như trong phương pháp động lượng, tức là $\mathbf{s}_t \leftarrow \gamma \mathbf{s}_{t-1} + (1-\gamma) \mathbf{g}_t^2$ cho các tham số $\gamma > 0$.
-Giữ nguyên tất cả các phần khác ta có thuật toán RMSProp.
+Vấn đề này có thể được giải quyết bằng cách sử dụng $\mathbf{s}_t / t$. 
+Đối với các phân phối hợp lý của $\mathbf{g}_t$, thuật toán sẽ hội tụ. 
+Đáng tiếc là có thể mất rất nhiều thời gian cho đến khi trạng thái giới hạn được bắt đầu, bởi thuật toán này ghi nhớ toàn bộ quỹ đạo của các giá trị. 
+Một cách khác là sử dụng trung bình rò rỉ tương tự như trong phương pháp động lượng, tức là $\mathbf{s}_t \leftarrow \gamma \mathbf{s}_{t-1} + (1-\gamma) \mathbf{g}_t^2$ cho các tham số $\gamma > 0$. 
+Giữ nguyên tất cả các phần khác ta có thuật toán RMSProp.
```
Comment (lkhphuc) [inline chapter_optimization/rmsprop_vn.md:42]:
```suggestion
Giữ nguyên tất cả các phần khác và ta có thuật toán RMSProp. 
```

File: chapter_optimization/rmsprop_vn.md [hunk 3]
```diff
@@ -224,10 +224,10 @@ d2l.train_gluon_ch11('rmsprop', {'learning_rate': 0.01, 'gamma1': 0.9},
 * The coefficient $\gamma$ determines how long the history is when adjusting the per-coordinate scale.
 -->
 
-* Thuật toán RMSProp rất giống với Adagrad ở chỗ cả hai đều sử dụng bình phương của gradient để chuyển đổi hệ số.
-* RMSProp có điểm chung với phương pháp động lượng là chúng đều sử dụng trung bình rò rỉ. Tuy nhiên, RMSProp sử dụng kỹ thuật này để điều chỉnh tiền điều kiện theo hệ số.
-* Trong thực tế, tốc độ học cần được định thời bởi người lập trình. 
-* Hệ số $\gamma$ xác định độ dài thông tin quá khứ được sử dụng khi điều chỉnh tỉ lệ theo từng toạ độ.
+* Thuật toán RMSProp rất giống với Adagrad ở chỗ cả hai đều sử dụng bình phương của gradient để chuyển đổi hệ số.
```
Comment (lkhphuc) [inline chapter_optimization/rmsprop_vn.md:227]:
```suggestion
* Thuật toán RMSProp rất giống với Adagrad ở chỗ cả hai đều sử dụng bình phương của gradient để thay đổi tỉ lệ hệ số. 
```

File: chapter_optimization/rmsprop_vn.md [hunk 4]
```diff
@@ -242,10 +242,10 @@ d2l.train_gluon_ch11('rmsprop', {'learning_rate': 0.01, 'gamma1': 0.9},
 4. Would you want to adjust $\gamma$ as optimization progresses? How sensitive is RMSProp to this?
 -->
 
-1. Điều gì sẽ xảy ra nếu ta đặt $\gamma = 1$? Giải thích tại sao?
-2. Biến đổi bài toán tối ưu thành cực tiểu hoá $f(\mathbf{x}) = 0.1 (x_1 + x_2)^2 + 2 (x_1 - x_2)^2$. Sự hội tụ sẽ diễn ra như thế nào?
-3. Hãy thử áp dụng RMSProp cho một bài toán học máy cụ thể, chẳng hạn như huấn luyện trên tập Fashion-MNIST. Hãy thí nghiệm với các cách điều chỉnh tốc độ học khác nhau.
-4. Bạn có muốn điều chỉnh $\gamma$ khi việc tối ưu tiến triển không? Hãy cho biết độ nhạy của RMSProp với việc điều chỉnh này?
+1. Điều gì sẽ xảy ra nếu ta đặt $\gamma = 1$? Giải thích tại sao? 
+2. Biến đổi bài toán tối ưu thành cực tiểu hoá $f(\mathbf{x}) = 0.1 (x_1 + x_2)^2 + 2 (x_1 - x_2)^2$. Sự hội tụ sẽ diễn ra như thế nào? 
+3. Hãy thử áp dụng RMSProp cho một bài toán học máy cụ thể, chẳng hạn như huấn luyện trên tập Fashion-MNIST. Hãy thí nghiệm với các cách điều chỉnh tốc độ học khác nhau.
```
Comment (lkhphuc) [inline chapter_optimization/rmsprop_vn.md:247]:
```suggestion
3. Hãy thử áp dụng RMSProp cho một bài toán học máy cụ thể, chẳng hạn như huấn luyện trên tập Fashion-MNIST. Hãy thí nghiệm với các tốc độ học khác nhau. 
```

File: chapter_optimization/rmsprop_vn.md [hunk 5]
```diff
@@ -242,10 +242,10 @@ d2l.train_gluon_ch11('rmsprop', {'learning_rate': 0.01, 'gamma1': 0.9},
...
 -->
 
-1. Điều gì sẽ xảy ra nếu ta đặt $\gamma = 1$? Giải thích tại sao?
-2. Biến đổi bài toán tối ưu thành cực tiểu hoá $f(\mathbf{x}) = 0.1 (x_1 + x_2)^2 + 2 (x_1 - x_2)^2$. Sự hội tụ sẽ diễn ra như thế nào?
-3. Hãy thử áp dụng RMSProp cho một bài toán học máy cụ thể, chẳng hạn như huấn luyện trên tập Fashion-MNIST. Hãy thí nghiệm với các cách điều chỉnh tốc độ học khác nhau.
-4. Bạn có muốn điều chỉnh $\gamma$ khi việc tối ưu tiến triển không? Hãy cho biết độ nhạy của RMSProp với việc điều chỉnh này?
+1. Điều gì sẽ xảy ra nếu ta đặt $\gamma = 1$? Giải thích tại sao? 
+2. Biến đổi bài toán tối ưu thành cực tiểu hoá $f(\mathbf{x}) = 0.1 (x_1 + x_2)^2 + 2 (x_1 - x_2)^2$. Sự hội tụ sẽ diễn ra như thế nào? 
+3. Hãy thử áp dụng RMSProp cho một bài toán học máy cụ thể, chẳng hạn như huấn luyện trên tập Fashion-MNIST. Hãy thí nghiệm với các cách điều chỉnh tốc độ học khác nhau. 
+4. Bạn có muốn điều chỉnh $\gamma$ khi việc tối ưu tiến triển không? Hãy cho biết độ nhạy của RMSProp với việc điều chỉnh này?
```
Comment (lkhphuc) [inline chapter_optimization/rmsprop_vn.md:248]:
```suggestion
4. Bạn có muốn điều chỉnh $\gamma$ khi việc tối ưu đang tiến triển không? Hãy cho biết độ nhạy của RMSProp với việc điều chỉnh này? 
```

File: chapter_optimization/rmsprop_vn.md [hunk 6]
```diff
@@ -72,10 +72,10 @@ Expanding the definition of $\mathbf{s}_t$ yields
 -->
 
 
-Hằng số $\epsilon > 0$ thường được đặt bằng $10^{-6}$ để đảm bảo rằng chúng ta sẽ không gặp vấn đề chia cho 0 hoặc kích thước bước quá lớn.
-Với khai triển này, bây giờ chúng ta có thể tự do kiểm soát tốc độ học $\eta$ độc lập với phép chuyển đổi được áp dụng cho từng tọa độ.
-Về mặt trung bình rò rỉ, chúng ta có thể áp dụng các lập luận tương tự như trước trong phương pháp động lượng.
-Khai triển định nghĩa $\mathbf{s}_t$ ta có
+Hằng số $\epsilon > 0$ thường được đặt bằng $10^{-6}$ để đảm bảo rằng chúng ta sẽ không gặp vấn đề khi chia cho 0 hoặc kích thước bước quá lớn. 
+Với khai triển này, bây giờ chúng ta có thể tự do kiểm soát tốc độ học $\eta$ độc lập với phép chuyển đổi được áp dụng cho từng tọa độ.
```
Comment (rootonchair) [inline chapter_optimization/rmsprop_vn.md:76]:
```suggestion
Với khai triển này, bây giờ chúng ta có thể tự do kiểm soát tốc độ học $\eta$ độc lập với phép biến đổi tỉ lệ được áp dụng cho từng tọa độ. 
```

File: chapter_optimization/rmsprop_vn.md [hunk 7]
```diff
@@ -13,19 +13,19 @@ While this is generally appropriate for convex problems, it might not be ideal f
 Yet, the coordinate-wise adaptivity of Adagrad is highly desirable as a preconditioner.
 -->
 
-Một trong những vấn đề then chốt trong :numref:`sec_adagrad` là tốc độ học giảm theo một định thời được định nghĩa sẵn $\mathcal{O}(t^{-\frac{1}{2}})$ một cách hiệu quả.
-Nhìn chung, cách này thích hợp với các bài toán lồi nhưng có thể không phải giải pháp lý tưởng cho những bài toán không lồi, chẳng hạn những bài toán gặp phải trong học sâu.
-Tuy vậy, khả năng thích ứng theo toạ độ của Adagrad là rất tuyệt vời cho một bộ tiền điều kiện (_preconditioner_).
+Một trong những vấn đề then chốt trong :numref:`sec_adagrad` là tốc độ học thực tế được giảm theo một định thời được định nghĩa sẵn $\mathcal{O}(t^{-\frac{1}{2}})$.
```
[the remark this answers is not in the document]
Comment (lkhphuc) [reply chapter_optimization/rmsprop_vn.md]: `thời điểm` đâu có cùng nghĩa với `scheduling` đâu nhỉ

[the remark this answers is not in the document]
Comment (rootonchair) [reply chapter_optimization/rmsprop_vn.md]: Câu gốc là:
> One of the key issues in :numref:`sec_adagrad` is that the learning rate decreases at a predefined schedule of effectively $\mathcal{O}(t^{-\frac{1}{2}})$.

thì em thấy nếu dịch sát nghĩa đọc bị rối ạ vì `định thời` gần với động từ hơn là danh từ. Nên em sửa lại theo cách hiểu nôm na của em
